Source/Core/Core/ConfigManager.cpp

```diff
@@ -1151,3 +1151,42 @@ std::vector<std::string> SConfig::GetGameIniFilenames(const std::string& id, u16
 
   return filenames;
 }
+
+bool SConfig::GCLanguageAndRegionCompatible() const
+{
+  switch (m_region)
+  {
+  case DiscIO::Region::NTSC_J:
+    return (SelectedLanguage == 0) && bSelectNTSCAsJapanese;  // Japanese
+
+  case DiscIO::Region::NTSC_U:
+    return (SelectedLanguage == 0) && !bSelectNTSCAsJapanese;  // English
+
+  case DiscIO::Region::PAL:
+    return ((SelectedLanguage == 0) && !bSelectNTSCAsJapanese) ||  // English
+           SelectedLanguage == 1 ||                                // German
+           SelectedLanguage == 2 ||                                // French
+           SelectedLanguage == 3 ||                                // Spanish
+           SelectedLanguage == 4 ||                                // Italian
+           SelectedLanguage == 5;                                  // Dutch
```

Comment:
Is it acceptable that the NTSC-J VI bit doesn't get forced off for languages 1-5?